aire/src/main/accordion/accordion-item.ts

```diff
@@ -0,0 +1,20 @@
+import {bindable, customElement, inject} from "aurelia-framework";
+import {dom} from "aire/core/dom";
+
+@inject(Element)
+@customElement('accordion-item')
+export class AccordionItem {
+
+    li      : HTMLElement;
```

Comment:
If you know the referenced element type, make sure that's reflected in the type of these variables (e.g. HTMLLiElement)